src/help/components/contextual/SideButtons.tsx

```diff
@@ -16,6 +17,36 @@ const SideButtons = ({ displayHelp, onClick }: Props) => {
   const [displayFeedback, setDisplayFeedback] = useState(false);
 
   useEffect(() => {
+    // Checking if there is a scroll bar
+    const mainContent = document?.querySelector<HTMLElement>(
+      `.${baseStyles['main-content']}`
+    );
+    const scrollBarWidth =
+      mainContent && mainContent.offsetWidth - mainContent.clientWidth;
+
+    if (scrollBarWidth) {
+      const hjButton = document.querySelector<HTMLElement>(
+        '#_hj_feedback_container div button'
+      );
```

Comment:
this should probably go lower, within the function line 50, because hotjar is added later and doing this too early you might not get a handle on the button yet

Reply:
I was under the impression that hotjar was added first, and if there's no hotjar even after `sleep(3000)` we then display the fallback feedback button? Anyways I did move it so it's underneath the `sleep` in the next commit

Reply:
The hotjar logic is running immediately, although on some browsers (probably slow ones) we noticed that there was a delay before adding the button. It would cause both or fallback feedback and the hotjar one to be loaded on top of each other, hence the extra 3000ms delay before checking